src/bit-systems/scene-loading.ts

```diff
@@ -81,7 +90,23 @@ function* loadScene(
     }
     environmentSystem.updateEnvironmentSettings(envSettings);
 
-    const sceneEl = AFRAME.scenes[0];
+    const cameraNode = (document.getElementById("scene-preview-node") as AElement).object3D!;
+    removeComponent(world, ScenePreviewCamera, cameraNode.eid!);
+    const sceneObj = world.eid2obj.get(scene)!;
+    const previewCamera = sceneObj.getObjectByName("scene-preview-camera");
```

Comment:
For future clean up, similar to the above. We may use query and component to access preview camera.